src/main/flight/mixer.c

```diff
@@ -69,6 +69,12 @@ PG_REGISTER_WITH_RESET_TEMPLATE(mixerConfig_t, mixerConfig, PG_MIXER_CONFIG, 0);
 #ifndef TARGET_DEFAULT_MIXER
 #define TARGET_DEFAULT_MIXER    MIXER_QUADX
 #endif
+
+#ifdef USE_DYN_LPF
```

Comment:
Not sure if it's necessary to make the defines conditional - their names make clashes and use outside of the intended scope unlikely.